### PR TITLE
[MRG+1] add plotting of a connectome using a surface atlas on fsaverage

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: examples/01_plotting/plot_surf_atlas.py [hunk 1]
```diff
@@ -77,6 +77,34 @@
                        darkness=.5)
 plotting.show()
 
+###############################################################################
+# Display connectome from surface parcellation
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_surf_atlas.py]: A little more text explaining what you are doing here would be useful for non experts

File: examples/01_plotting/plot_surf_atlas.py [hunk 2]
```diff
@@ -77,6 +77,34 @@
...
+labels = destrieux_atlas['labels']
+for hemi in ['left', 'right']:
+    vert = destrieux_atlas['map_%s' % hemi]
+    rr, _ = surface.load_surf_mesh(fsaverage['pial_%s' % hemi])
+    for k, label in enumerate(labels):
+        if "Unknown" not in str(label):
+            coordinates.append(np.mean(rr[vert == k], axis=0))
+
+coordinates = np.array(coordinates)
+
```
Comment (GaelVaroquaux) [inline examples/01_plotting/plot_surf_atlas.py]: Here are comment saying what your doing

**Comment by jeromedockes** [comment]
ci failure is due to a neurosynth server error and unrelated to this PR

**Comment by kchawla-pi** [comment]
How long do its server outages usually last?

**Comment by agramfort** [comment]
comments added

**Comment by GaelVaroquaux** [comment]
LGTM. +1 for merge. Thank you!

**Comment by kchawla-pi** [comment]
Hey please do not merge it right away, I'm changing circle CI and need all
the containers and the master untouched.

**Comment by GaelVaroquaux** [comment]
Merging, now!

**Pull request merged**